app/assets/stylesheets/lifehack_style.scss

```diff
@@ -0,0 +1,160 @@
+.form-text {
+  float: left;
+  font-size: 15px;
+  font-weight: bold;
+  margin: 15px;
+}
+
+.lifehack-details {
+  max-width: 300rem;
+  padding: rem-calc(50);
+}
+
+.panel {
+
+  h5 {
+    font-size: 30px;
+    text-align: justify;
+    padding: 23px;
+  }
+
+  h2 {
+    border-style: double;
+    color: $primary-color;
+    font-size: 30px;
+    padding:  40px 2.875rem 40px 2.875rem;
+    text-align: left;
+    }
+
+  h3 {
+    color: $black;
+    font-size: 30px;
+    padding:  9px 2.875rem 10px 2.875rem;
+    text-align: left;
+  }
+
+  table tr td {
+    background-color: $smoke;
+    padding: 13px 0.875rem 13px 2.875rem;
+  }
+
+  .creator-name {
+    padding: 0;
+  }
+
+  .creator-image {
+    padding: 5;
+  }
+
+  .lifehack-description {
+    font-size: 20px;
+  }
+}
+
+.description {
+
+  th {
+    background-color: $smoke;
+    font-size: 18px;
+    margin-left: auto;
+    margin-right: auto;
+    padding: 3px 5.875rem 21px 4.875rem;
+    text-align: center;
+  }
+
+  td {
+
+    background-color: $smoke;
+    font-size: 15px;
+    margin-left: auto;
+    margin-right: auto;
+    padding: 1px 3.875rem 1px 2.875rem;
+    text-align: center;
+
+  }
+}
+
+.show-descrip tr{
+    color: $primary-color;
+    font-size: 30px;
+    padding: 30px 1.875rem 30px 1.875rem;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Shorthand form for property `padding` should be written more concisely as `30px 1.875rem` instead of `30px 1.875rem 30px 1.875rem`